bls/src/main/java/tech/pegasys/teku/bls/BLS.java

```diff
@@ -130,6 +138,10 @@ public static boolean aggregateVerify(
    */
   public static boolean fastAggregateVerify(
       List<BLSPublicKey> publicKeys, Bytes message, BLSSignature signature) {
+    if (!BLSConstants.VERIFICATION_ENABLED) {
+      LOG.warn("Skipping bls verification.");
```

Comment:
BTW wouldn't there be a flood of `WARN` logs when fuzz testing?

Reply:
I think that's ok. We want to avoid a worst-case scenario in which somebody accidently runs without signature verification in production. Making it very noisy should help with this. I expect that fuzzers should ignore log lines, at least below ERROR level.